api/app.go

```diff
@@ -217,7 +218,23 @@ func (app *App) loadConfiguration() error {
 }
 
 func (app *App) configureEnrichment() {
-	enricher := enriching.NewEnricher(app.ParsedConfig.Enrichment, app.Logger)
+	// If a specific Redis instance is not provided, use the same one as the rest of the service.
+	if app.ParsedConfig.Enrichment.Cache == nil {
+		app.ParsedConfig.Enrichment.Cache = &enriching.Cache{RedisOptions: app.ParsedConfig.Redis}
```

Comment:
If no cache configuration is provided, the system should work without any caching instead of trying to force the Redis database as a cache database.

In addition to the `EnricherCache` implementation you provided using Redis, you can implement a `NoOpEnricherCache` that does nothing and is used when no cache configuration is provided.

Reply:
Since cache is optional, I would say we need a `Enabled` configuration, not `Disabled`. The default is not having cache.

I am not aware of the Cloud Save stuff, but it seems similar. Maybe should we change `Disabled` to `Enabled`?

Reply:
In fact, better thinking about this, we should not have neither `Enabled` nor `Disabled`. If the configuration is not set, we do not activate it.

Reply:
In both cases I used `Disabled` to make it enabled by default, with the option to disable it. Do you think that's a bad way to do it?